code/datums/commodity.dm

```diff
@@ -659,47 +659,47 @@
 
 /datum/commodity/drugs/poppies
 	comname = "Poppies"
-	comtype = /obj/item/plant/herb/poppy
```

Comment:
doesn't this prevent them from buying them, however?

Reply:
I'd imagine `comtype = /obj/item/plant/herb/poppy/spawnable`, the line replacing the highlighted line, should work? I had difficulties getting RNG to comply during tests but for the cannabis and mushrooms I had the expected behavior.

Reply:
doesn't that make it so they can only buy the spawnable variety?

Reply:
Correct, the spawnable variety just comes with morphine, whereas the regular version doesn't. From my understanding this is consistent with the rest of the changes I made; I assume I overlooked something?

Reply:
I mean the players selling to the trader. They will only be able to sell the spawnable variant to them, of which they cannot farm themselves.

Reply:
OH, fuck okay, good catch, thank you!! I'll fix that soon; just need to make a new spawnable variant of the drugs and slap those into the sold versions.